### ISSUE TITLE
set_learning_rate crash!

### ISSUE THREAD

**Issue by xinsuinizhuan** [josephjaspers/blackcat_tensors#66]
When i call the set_learning_rate(0.001) to set learning_rate, it crash!
![图片](https://user-images.githubusercontent.com/40679769/80666000-9ae74d00-8acd-11ea-984d-5b001f10462b.png)

**Comment by xinsuinizhuan** [comment]
it seems lstm.h set_learning_rate_hook.
I don't set learning rate, then to train, when i done, and to save the modle, it also crash:
![图片](https://user-images.githubusercontent.com/40679769/80667357-8a38d600-8ad1-11ea-929f-674a05628eee.png)

**Comment by josephjaspers** [comment]
Hello! I am sorry its been a while! 
I will be looking into these when I can!

I am alive and well. I hope to be able to start working on this again in the near future.
I will most likely move the neural networks to a seperate project, but I will maintain both of them!

Sorry for the long delay!

**Comment by xinsuinizhuan** [comment]
> 
> 
> Hello! I am sorry its been a while!
> I will be looking into these when I can!
> 
> I am alive and well. I hope to be able to start working on this again in the near future.
> I will most likely move the neural networks to a seperate project, but I will maintain both of them!
> 
> Sorry for the long delay!

You are well, it is a great good new. I am so worried about you.  I am also pay attention to you, you updates your project and star others projection, so i sometimes think you should alive. Protect yourself, stay home when necessary.
